web/css/main.css

```
@@ -213,74 +339,1225 @@ body.pc-vertical-stretch {
+    @apply py-4;
+    @apply sm:px-4;
+    @apply sm:py-5;
+    @apply sm:p-6;
+}
+
+.pc-widget-host {
+    @apply relative;
+    @apply mx-auto;
+    @apply mt-2;
+    @apply w-full;
+    width: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)